.github/workflows/update-styles.yml

```diff
@@ -0,0 +1,55 @@
+name: Weekly Styles Update
+on:
+  workflow_dispatch:
+    inputs:
+      name:
+        description: 'Person who triggered the workflow'
+        required: true
+        default: 'Maintainer (via GitHub)'
+  schedule:
+    # Run every Wednesday
+    - cron: 0 0 * * 3
+
+jobs:
+  update-styles:
+    runs-on: ubuntu-20.04
+    name: Styles Update
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Record Date
+        id: today
+        run: |
+          echo "::set-output name=date::$(date -I)"
+          echo "::set-output name=min::$(date +%Y-%m-%d-%H-%M)"
+
+      - name: Sync lesson with carpentries/styles
+        id: update
+        uses: zkamvar/actions/update-styles@main
```

Comment:
would be nice to transfer this repo (or just the update-styles action) to the carpentries org.

Reply:
Oh! Thank you for pointing this out! It is done!